src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -512,6 +512,18 @@ public void EmptyEnumStoreTest()
 
             Assert.True(enums.Count() > 0);
         }
+        
+        /// <summary>
+        /// Issue #263 - Filter should be able to handle empty rows
+        /// </summary>
+        [Fact]
+        public void BlankRowFilter()

```

Comment:
>BlankRowFilter [](http://example.com/codeflow?start=20&length=14)

Can you add this as a test case here instead? 
The .txt cases are easier to manage

[https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions.txt](https://github.com/microsoft/Power-Fx/blob/main/src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions.txt)

Those .txt are run through a special runner here: [https://github.com/microsoft/Power-Fx/blob/4e57e10da30e19025b9c35885bfb99571404639f/src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs#L24](https://github.com/microsoft/Power-Fx/blob/4e57e10da30e19025b9c35885bfb99571404639f/src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs#L24)